src/PostOffice.php

```diff
@@ -0,0 +1,155 @@
+<?php
+
+namespace CanadaPost;
+
+class PostOffice extends ClientBase
+{
+    /**
+     * Get information on Post Offices nearest to a given postal code.
+     *
+     * @param string $postal_code
+     *   The postal code.
+     * @param bool $d2po
+     *   True indicates that you want a list of Post Offices that accept the
+     * "Deliver to Post Office" delivery of parcels. Use this when you want the
+     * parcel to be delivered directly to a Post Office rather than to the
+     * recipient’s address
```

Comment:
so i was looking up some code styles for phpdocblocks:

- https://pear.php.net/manual/en/standards.sample.php
- http://docs.phpdoc.org/references/phpdoc/basic-syntax.html

 seems like maybe we should switch to:

```php
    /**
     * Get information on Post Offices nearest to a given postal code.
     *
     * @param string $postal_code The postal code.
     * @param bool   $d2po        True indicates that you want a list of Post
     *                            Offices that accept the "Deliver to Post
     *                            Office" delivery of parcels. Use this when
     *                            you want the parcel to be delivered directly
     *                            to a Post Office rather than to the
     *                            recipient’s address
     * @param int    $max_addresses
     *                            The maximum number of Post Offices to return
     *                            with the response. The maximum allowed is 50.
     *                            In remote locations, fewer Post Offices than
     *                            requested may be returned.
     * @param array  $options
     *                            The options array.
     *
     * @return \DOMDocument|\Psr\Http\Message\StreamInterface
     * @throws \GuzzleHttp\Exception\GuzzleException
     */
    public function getNearestPostOffice(
        $postal_code,
        $d2po = false,
        $max_addresses = 10,
        array $options = []
    ) {...}
```

Reply:
This is commonly used but it is problematic because you then either have to not respect line length and lose on readability or to have 20-30 character lines in some cases, which is just as bad. I like the way it's done in Drupal, starting the description on a new line.